src/background.js

```diff
@@ -776,18 +776,56 @@ function onMessageHandler(request, sender, callback) {
 		panelData.set(message);
 		callback();
 		return false;
+	} else if (name === 'account.getTheme') {
+		if (message.currentTheme !== 'default' &&
+			account.hasScopesUnverified(['subscription:supporter'])) {
+			// try to get it locally
+			message.theme = (conf.themes || {})[message.currentTheme];
+			if (message.theme) {
+				// This will trigger panelData.set through dispatch
+				// as currentTheme is in SYNC_ARRAY
+				conf.current_theme = message.currentTheme;
+				callback(message);
+				return false;
+			}
+			account.getTheme(`${message.currentTheme}.css`).then((theme) => {
+				if (theme) {
+					const themes = conf.themes || {};
```

Comment:
@zarembsky can we set `conf.themes` to `null` instead of an empty object when not set?

Reply:
No, if we look at the next line:
```const themes = conf.themes || {};
themes[message.currentTheme] = theme;```